README.md

```diff
@@ -158,27 +158,40 @@ And define our draw() function:
                 serverUser: "neo4j",
                 serverPassword: "sorts-swims-burglaries",
             },
+            visConfig: {
+                nodes: {
+                    shape: 'dot'
+                },
+                edges: {
+                    arrows: {
+                        to: {enabled: true}
+                    }
+                },
+            },
             labels: {
                 Character: {
                     label: "name",
                     value: "pagerank",
                     group: "community",
                     [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
                         function: {
-                            title: (node) => viz.nodeToHtml(node, [
-                                "name",
-                                "pagerank"
-                            ])
+                            title: (node) => node.properties.name
```

Comment:
This is wrong, the point of the title is to be popup with all the data same for the relationship

Reply:
When I ran the version with viz.nodeToHtml it throws an error. Is it just for me? Could you check it?

Reply:
Cam you send here traceback?

Reply:
```
index.html:62 Uncaught (in promise) TypeError: viz.nodeToHtml is not a function
    at title (index.html:62:51)
    at RB.BB (neovis.js@2.1.0:2:1153480)
    at RB.ZB (neovis.js@2.1.0:2:1154335)
    at ZB.next (<anonymous>)
    at Function.all (<anonymous>)
    at RB.UB (neovis.js@2.1.0:2:1154913)
    at async RB.VB (neovis.js@2.1.0:2:1155187)
    at async neovis.js@2.1.0:2:1156717
    at async Promise.all (index 0)
```

Reply:
Also, if the person is using react he won't find this function in the package (my case).

Reply:
Yeah, it should be NeoVis.objectToHtml (on the class, not on the instance (make sure you are on latest as on 2.0.2 it was imported separatly)

Reply:
I'm using version 2.1.0. Still

```
index.html:62  Uncaught (in promise) TypeError: NeoVis.nodeToHtml is not a function
    at title (index.html:62:54)
    at RB.BB (neovis.js@2.1.0:2:1153480)
    at RB.ZB (neovis.js@2.1.0:2:1154335)
    at ZB.next (<anonymous>)
    at Function.all (<anonymous>)
    at RB.UB (neovis.js@2.1.0:2:1154913)
    at async RB.VB (neovis.js@2.1.0:2:1155187)
    at async neovis.js@2.1.0:2:1156717
    at async Promise.all (index 0)
title @ index.html:62
BB @ neovis.js@2.1.0:2
ZB @ neovis.js@2.1.0:2
UB @ neovis.js@2.1.0:2
```

Reply:
How are you importing neovis?

Reply:
How should I import? This error is on a pure html file...

```
<div id="viz"></div>
<script src="https://unpkg.com/neovis.js@2.1.0"></script>

<script type="text/javascript">
    window.onload = () => {
        draw();
    }

    let neoViz;

    // https://visjs.github.io/vis-network/docs/network
    function draw() {
        const config = {
            containerId: "viz",
            neo4j: {
                serverUrl: "bolt://localhost:7687",
                serverUser: "neo4j",
                serverPassword: "abacaxi1234",
            },
            visConfig: {
                nodes: {
                    shape: 'circularImage', //dot, circle, image, circularImage
                },
                edges: {
                    arrows: {
                        to: {enabled: true}
                    }
                },
            },
            labels: {
                Character: {
                    label: "name",
                    value: "pagerank",
                    group: "community",
                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
                        function: {
                             title: (node) => NeoVis.nodeToHtml(node, [
                                "name",
                                "pagerank"
                            ]),
                            image: (node) => 'https://th.bing.com/th/id/R.daa762238acb5391d760f15512430c93?rik=1RaEtbM%2fwS4y5w&riu=http%3a%2f%2fantoniocorreia.smugmug.com%2fphotos%2f398317216_WxpWA-M.jpg&ehk=Lm4s5VQYoT7GySHeYukp6krp5hjYjZsW%2bMgngedUpbA%3d&risl=&pid=ImgRaw&r=0',
                            // label: (node) => null,
                        }
                    }
                }
            },
            relationships: {
                INTERACTS: {
                    value: "weight"
                }
            },
            initialCypher: "MATCH (n)-[r:INTERACTS]->(m) RETURN n,r,m LIMIT 100"
        };

        neoViz = new NeoVis.default(config);
        neoViz.render();
    }
</script>
```

Reply:
NeoVis.objectToTitleHtml

Reply:
I renamed it in 2.x and forgot 😅